### DISCUSSION TITLE
Propose for a component like "treetable"

### DISCUSSION THREAD

**Discussion by donald001** [quasarframework/quasar#16850] [CLI - PWA mode] [answered]
Both tree and table are excellent components in Quasar, but sometimes we want a combination of them both. So, it would be  better if quasar can provide such a component that displays data in a table, but the data structure is a tree. Like this: https://primevue.org/treetable/
![image](https://github.com/quasarframework/quasar/assets/38281146/085295e3-ecb6-4d68-8b81-3898f54d048e)

**Comment by 549393092** [comment, marked as answer]
please check #4099
